build/common.sh

```diff
@@ -663,12 +663,9 @@ function kube::build::sync_to_container() {
   kube::build::rsync \
     --delete \
     --filter='H /.git' \
-    --filter='- /.make/' \
     --filter='- /_tmp/' \
     --filter='- /_output/' \
     --filter='- /' \
-    --filter='H zz_generated.*' \
-    --filter='H generated.proto' \
```

Comment:
We want to start syncing these in now?

Reply:
yes.  I admit that some of this file scares me still, but I am sure we want to sync them in.